cmd/ipfs/init.go

```diff
@@ -90,6 +94,12 @@ environment variable:
 			return
 		}
 
+		keyType, _, err := req.Option("k").Int()
```

Comment:
I used the short option name for consistency. See `empty, _, err := req.Option("e").Bool()` and `nBitsForKeypair, _, err := req.Option("b").Int()` right above.